cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -14,32 +14,11 @@
  * limitations under the License.
  */
 
-#include <errno.h>
-#include <string.h>
```

Comment:
Most of the OS headers were not moved, guess this was intentional?

Reply:
Done.